src/index.ejs

```diff
@@ -9,6 +9,14 @@
     <title>Pasty</title>
   </head>
   <body>
+    <noscript>
+      <h1>Javascript Disabled</h1>
+      <p> You appear to have Javascript disabled. <br />
+        Javascript is <strong>required</strong> to use this website. <br />
+        Pasty makes use of client-side encryption and decryption, which necessitates Javascript. <br />
+        <br />
+        The Javascript which will be run is available, under an open source license, <a href="https://github.com/brhoades/pasty-http-client">here</a>.
```

Comment:
I've recently renamed the project to pasty-react. Could you update this url to:

https://github.com/brhoades/pasty-react